workspace/packages/local/Deck/package.json

```diff
@@ -45,14 +45,11 @@
         ]
     },
     "js": [{
-        "path": "resources/lib/underscore/underscore-min.js",
-        "bundle": true
+        "path": "resources/lib/underscore/underscore-min.js"
     }, {
-        "path": "resources/lib/Marked/marked.js",
-        "bundle": true
+        "path": "resources/lib/Marked/marked.js"
     }, {
-        "path": "resources/lib/String/string.min.js",
-        "bundle": true
+        "path": "resources/lib/String/string.min.js"
```

Comment:
There's no need for "bundle": true here.
"bundle" tells Sencha Cmd which file contains the whole framework + app when the app is build. These are package resources, so this setting does not apply here.